### ISSUE TITLE
Single tunnel level

### ISSUE THREAD

**Issue by rexim** [rexim/beatwave#11]
### Description

Design a level with a simple drum sequence using only tunnel turns. Adjust all the transition times as needed.

**Comment by Minoru** [comment]
It's blocked by #8, right? Right?

**Comment by rexim** [comment]
Well, I believe it's not. The scope of this issue is to design just one simple sample level with tunnels idea and see if it works. It's completely orthogonal to #8 :)

**Comment by Minoru** [comment]
Okay ._.

**Comment by rexim** [comment]
@Minoru no worries my friend! There will be Haskell soon. :)

**Comment by rexim** [comment]
I believe it's already done
